### ISSUE TITLE
Diff file generation fails when actual and baseline pdfs has different number of pages

### ISSUE THREAD

**Issue by AJ-72** [marcdacz/compare-pdf#27]
Thank you for developing and open sourcing this project @marcdacz !

When the actual pdf is a single page PDF and baseline has more than one, the png diff generation fails. 
From the code I could see that when the image is generated, we are not adding any index (-0) for a single page pdf. 

**native.js file**
<img width="467" alt="native js" src="https://user-images.githubusercontent.com/4033097/195366246-6386490a-b15b-4d6f-9314-5c84572eaedc.png">

However, in compareimage.js we are look at the number of images in **baseline** and use the same suffix for actual. Due to this mismatch, we send the wrong actual file name to diff generator method which fails saying "couldn't find the actual file". 

**compareimage.js**
<img width="464" alt="compareimage js" src="https://user-images.githubusercontent.com/4033097/195367073-30d1b892-2ab6-40f5-aa0e-5009063204f4.png">

I could get this solved by making this change in compareimage.js.  Happy to raise a PR if this looks good.

<img width="540" alt="image" src="https://user-images.githubusercontent.com/4033097/195368300-e0760eba-ab0c-438b-8aa5-8bac681cb9ca.png">

**Comment by marcdacz** [comment]
@AJ-72 yeah mate I encourage you to raise a PR. Please add tests too just to be sure. Let me know how it goes

**Comment by AJ-72** [comment]
@marcdacz I have opened a PR for this fix with the change and a test to validate
